flow/project/cli/config/config.go

```diff
@@ -133,10 +135,14 @@ func (c *Contracts) GetByNetwork(network string) Contracts {
 }
 
 // GetAccountByName get account by name
-func (a *Accounts) GetByName(name string) Account {
-	return funk.Filter([]Account(*a), func(a Account) bool {
-		return a.Name == name
-	}).([]Account)[0]
+func (a *Accounts) GetByName(name string) *Account {
+	for _, account := range *a {
+		if account.Name == name {
+			return &account
+		}
+	}
+
+	return nil
```

Comment:
Nice. Even though it's more code, I think this is easier to read

Reply:
Yes, you were right. I will change all of them to be this way.